src/netstandard/ref/System.Data.cs

```diff
@@ -180,6 +180,40 @@ public sealed partial class DataColumnCollection : System.Data.InternalDataColle
         public void Remove(string name) { }
         public void RemoveAt(int index) { }
     }
+    public static partial class DataReaderExtensions
+    {
+        public static bool GetBoolean(this System.Data.Common.DbDataReader reader, string name) { throw null; }
+        public static byte GetByte(this System.Data.Common.DbDataReader reader, string name) { throw null; }
+        public static long GetBytes(this System.Data.Common.DbDataReader reader, string name, long dataOffset, byte[] buffer, int bufferOffset, int length) { throw null; }
+        public static char GetChar(this System.Data.Common.DbDataReader reader, string name) { throw null; }
+        public static long GetChars(this System.Data.Common.DbDataReader reader, string name, long dataOffset, char[] buffer, int bufferOffset, int length) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        public static System.Data.Common.DbDataReader GetData(this System.Data.Common.DbDataReader reader, string name) { throw null; }
```

Comment:
What is the reason for adding API to the standard when it should be always hidden to the developers?

Reply:
@marek-safar how do you mean, hidden? These are public APIs under System.Data which are meant to be consumed...

Reply:
It has [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))] which usually used to hide the API from the developers

Reply:
Oh I see.

The same attribute is on [the original DbDataReader.GetData()](https://github.com/dotnet/standard/blob/master/src/netstandard/ref/System.Data.Common.cs#L478) which accepts an int - I copied that entire API surface for consistency, and the attribute came with that.

It's true that this API is extremely old and probably non-useful - we could not introduce the string-based overload for it. @divega unless you have some more insight or another opinion I will make this change.

Reply:
It is an *interesting* API to get nested data readers. I don't know what ADO.NET providers implement it. 

But presumably the string-based extension method should have the same value, relative to the int instance method, as all the other extension methods we are adding. 

Given the string-based method is already in .NET Core 3.0, that the original int instance method is already part of the standard (with the attribute), that is not the only method in the class that has the attribute (e.g. Dispose and GetEnumerator have it as well), I am leaning towards adding this.

Reply:
At the very least we can say we're adding it for consistency, i.e. that the list of ordinal-accepting getters is identical to the list of string-accepting getters. @marek-safar and others, please let us know if you have a problem with keeping this overload.

Reply:
Agreed.